assets/app/view/game/corporate_buy_shares.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/button/buy_share'
+
+module View
+  module Game
+    class CorporateBuyShares < Snabberb::Component
+      include Actionable
+
+      needs :game, store: true
```

Comment:
this is not necessary because it's a part of actionable

Reply:
Removed and verified